### DISCUSSION TITLE
Sponsor brand report generate

### DISCUSSION THREAD

**Discussion by utsavvasoya** [amzn/ads-advanced-tools-docs#99] [Q&A] [answered]
I am not able to generate sponsor brand reports for report type "sponsoredBrandsCampaigns" can you please give me api endpoint so I can check it and generate a report for the sponsor brand. And also I need sales and spend data in that API response.

**Comment by elizafarley** [comment, marked as answer]
To generate a Sponsored Brands campaign report, you can use POST /v2/hsa/campaigns/report. A sample call (including spend and sales metrics) should resemble:

```curl
curl --request POST 'https://advertising-api.amazon.com/v2/hsa/campaigns/report' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer xxxxxx' \
--header 'Amazon-Advertising-API-ClientId: xxxxxx' \
--header 'Amazon-Advertising-API-Scope: xxxxxx' \
--data-raw '{
"reportDate": "20230622",
"metrics": "impressions,clicks,cost,attributedConversions14d,attributedSales14d",
"creativeType": "all"
}'
```

See the [full documentation](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v2/report-types#campaign-reports) for more info.

**Reply by utsavvasoya** [reply]
thanks for your reply.
